static/css/impala/fxa-migration.less

```diff
@@ -1,3 +1,13 @@
 .show-fxa-migration-banner #fxa-migration-banner {
     display: block;
 }
+
+.login-source-form .login-field,
+.login-source-form .login-button,
+.login-source-form #recap-container {
```

Comment:
Yeah, I wanted to avoid nesting things since this is pretty simple though. I could go either way but I think I prefer it this way for now.